contracts/BodhiCandy.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.4.18;
+
+import './libs/SafeMath.sol';
+
+contract BodhiCandy {
+    using SafeMath for uint256;
+
+    // Amount in Satoshi to deposit
+    uint256 public constant depositAmount = 1000000;
+
+    // Amount of blocks it takes for the last person to deposit to win if no others deposit within the length.
+    uint256 public constant winningBlockLength = 30;
+
+    address public lastDepositer;
+    uint256 public lastDepositBlock;
+    uint256 public currentBalance;
```

Comment:
So we don't need to define static methods for `lastDepositBlock` and `lastDepositBlock`?

Reply:
no, public methods automatically create getters.